src/watts/plugin.py

```diff
@@ -72,7 +72,7 @@ def postrun(self, params, name) -> Results:
     def plugin_name(self):
         return type(self).__name__[6:]
 
-    def __call__(self, params: Parameters = None, name: str = 'Workflow', **kwargs) -> Results:
+    def __call__(self, params: Parameters = None, name: str = '', verbose=True, **kwargs) -> Results:
         """Run the complete workflow for the plugin
```

Comment:
it would be important to add showcase the use of 'name' option in the documentation, and add (or modify) an example using this capability.

Reply:
Thanks for the feedback. I've made some updates in the documentation to address this as well as in the MCNP example.